src/main/scala/content/ContentDao.scala

```diff
@@ -0,0 +1,44 @@
+package content
+
+import awscala.s3.{Bucket, S3}
+import cats.implicits._
+import com.typesafe.scalalogging.Logger
+import config.Config
+import response.ErrorResponse
+import response.ErrorResponse.InternalError
+
+import scala.util.{Failure, Try}
+
+trait ContentDao {
+  def storeContents(contents: List[Content]): Either[ErrorResponse, Unit]
+
+  def deleteContent(key: String): Either[ErrorResponse, Unit]
+
+  def getContentCount: Either[ErrorResponse, Int]
+}
+
+class S3ContentDao(s3: S3) extends ContentDao {
+  val logger: Logger = Logger(classOf[S3ContentDao])
+  val bucketName: String = Config.getProperty("aws.content.bucket")
+  val bucket: Bucket = Bucket(bucketName)
```

Comment:
```suggestion
  implicit val s3: S3 // Some sort of implicit val so method calls don't require partial functions?
  val bucket: Bucket = Bucket(bucketName)
```
Usage of this bucket requires an implicit s3, but I pass in the S3 client as a form-of dependency injection. How can I use implicits within this context?